packages/common/src/utils/wallet.ts

```diff
@@ -134,3 +133,26 @@ export const convertWeiToWAudio = (amount: BN) => {
   const decimals = WEI_DECIMALS - SPL_DECIMALS
   return amount.div(new BN('1'.padEnd(decimals + 1, '0')))
 }
+
+/** USDC Utils */
+const BN_USDC_WEI = new BN('1000000')
+
+/** Formats a USDC wei string (full precision) to a fixed string suitable for
+display as a dollar amount. Note: WILL lose precision by rounding to nearest cent */
+export const formatUSDCWeiToUSDString = (amount: StringUSDC, precision = 2) => {
```

Comment:
Oh yeah, that's easy. I think if we're doing rounding, we need to process it a little ahead of time.

Reply:
I updated the code to add a rounding utility (since we will probably need that for some of the saga work) and then use the utility for formatting commas. Current behavior is to round up to the nearest cent and display that. We can discuss with Product and come back to change it later if needed.

Reply:
Yeah despite what I said in standup let's double check with product